### PR TITLE
Happy Thought - Jessica H

### PR DESCRIPTION
**Pull request by Jsfrulle**
https://a-happy-app.netlify.app/

### CODE REVIEW

File: code/src/AddDataToList.js [hunk 1]
```diff
@@ -0,0 +1,118 @@
+import React, { useState, useEffect } from "react";
+import List from "List";
+import Create from "Create";
+import Loading from "Loading";
```
Comment (DALA746) [inline code/src/AddDataToList.js:4]: Maybe naming components more descriptive?

File: code/src/AddDataToList.js [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+		Fetch();
+	}, []);
+
+	const Fetch = () => {
+		setLoading(true);
+		fetch(API)
+			.then((res) => res.json())
+			.then((data) => setPosts(data))
+			.finally(() => setLoading(false));
+	};
```
Comment (DALA746) [inline code/src/AddDataToList.js:31]: Naming function with uppercase letter is not a correct way, only components should have uppercase letters. So instead writing Fetch(), it should be fetch()

File: code/src/AddDataToList.js [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+
+
+
+/*  onClick event from message-box "Create" */
+
+	const handleSubmitMessages = (e) => {
+		e.preventDefault();
+		e.target.reset()
```
Comment (DALA746) [inline code/src/AddDataToList.js:39]: I wonder what does this line of code? Does it clean the input field? :)

File: code/src/App.js [hunk 4]
```diff
@@ -1,9 +1,29 @@
 import React from 'react'
+import AddDataToList from 'AddDataToList'
+import { Helmet } from 'react-helmet'
```
Comment (DALA746) [inline code/src/App.js:3]: What does this library do? :)

File: code/src/index.css [hunk 5]
```diff
@@ -1,13 +1,320 @@
...
+	0px 0px 50px rgb(255, 217, 1),
+	0px 0px 50px #ff0f0f,
+	0px 0px 150px #ff0f0f,
+	0px 10px 100px #ff0f0f,
+	0px 10px 100px #ff0f0f,
+	0px 10px 100px #ff0f0f,
+	0px 10px 100px #fbff00,
+	0px -10px 100px #ffd900,
+	0px -10px 100px #ff9900;
+}
```
Comment (DALA746) [inline code/src/index.css:293]: Why do you have so much code here? Is it code for loading the heart? :)

File: code/src/index.css [hunk 6]
```diff
@@ -1,13 +1,320 @@
...
+	word-break: normal;
+	padding: 10px 15px;
+	font-size: 15px;
+	font-weight: 700;
+}
+
+/* Text-box message */
+
+
+.TextPost{
```
Comment (DALA746) [inline code/src/index.css:191]: It is not necessary to name classes with uppercase, I believe only components should be named with uppercase.

**Review by DALA746 — COMMENTED** [review]
Hello Jessica! Great job on your project! Your project looks great and everything works as it should. I especially like your loading heart icon while API loading and the exclamation mark
inside of input, really creative! Your code looks a bit unstructured. Try to structure it in a better way next time. It is a lot of space between lines, tags are on different levels from each other both in JS and CSS so it is a bit hard to read the code.  Try to add comments to your code so you can explain the code to yourself and others :) ReadMe is not updated and I would like to hear your thoughts about this project. I couldn't find a link to the project in readme, so include all the links as well next time. 
If you have questions, don't hesitate to reach out to me :) 

Otherwise, I hope this project was educational for you and I am looking forward to seeing more of your projects in the future. Nice job! 💯

File: code/src/index.css (hunk 6 above)
Comment (Jsfrulle) [reply code/src/index.css:191]: Thanks for a superb feedback 🦾 that help me improve ! And Im going to implement name correct(I totally missed out on how to do it right) and be better to comment my code! I have a problem with prittier that im trying to fix(oh my code spacing funky!!) 

Thanks :D

**Pull request closed without merging**